src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -116,34 +124,35 @@
               (reset! selected (vec (vals selected-images)))
               (reset! selected @temporary-selected)))
           [selected-album])
-         [safe-area/consumer
-          (fn [insets]
+         [bottom-sheet-screen/consumer
+          (fn [close scroll-enabled on-scroll]
             (let [window-width       (:width (rn/get-window))
                   camera-roll-photos (rf/sub [:camera-roll/photos])
                   end-cursor         (rf/sub [:camera-roll/end-cursor])
                   loading?           (rf/sub [:camera-roll/loading-more])
                   has-next-page?     (rf/sub [:camera-roll/has-next-page])]
-              [rn/view {:style {:flex 1}}
+              [:<>
                [rn/view
                 {:style style/buttons-container}
-                (when platform/android?
-                  [rn/touchable-opacity
-                   {:active-opacity 1
-                    :on-press       #(rf/dispatch [:navigate-back])
-                    :style          (style/close-button-container)}
-                   [quo/icon :i/close
-                    {:size 20 :color (colors/theme-colors colors/black colors/white)}]])
-                [album-title true selected-album selected temporary-selected]
+                [rn/touchable-opacity
```

Comment:
isn't this like top-nav component?

Reply:
It is a close button. Do we have a reusable one?

Reply:
yeah we have it inside top-nav component, i wanted to check it in figma but i don't see the button there 

<img width="527" alt="image" src="https://user-images.githubusercontent.com/11790366/225880887-d32d0f9a-9f8e-430a-af25-3783d0247174.png">

Reply:
also recent looks different, btw this down arrow is confusing, but its question for designers

Reply:
@flexsurfer Yeah there is no button. It was added previously to not block android users without a back button. But I think it can be removed now. 

I see some bottom-sheets have a close button some don't. I think it would be better if we are consistent on that. I will check with Pedro.

The `recent` button currently looks like when it has scrolled. It needs to animate to a blurry button after scrolling. There is an open issue: https://github.com/status-im/status-mobile/issues/14950

Reply:
Quoting Pedro:
"so on dekstop the ones with the X will be modals
and the ones without will be popover"